src/OmniSharp.Roslyn/Services/AbstractCodeActionProvider.cs

```diff
@@ -5,19 +5,39 @@
 using System.Reflection;
 using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.Extensions.Logging;
 
 namespace OmniSharp.Services
 {
     public abstract class AbstractCodeActionProvider : ICodeActionProvider
     {
+        private readonly bool _throwOnException = true;
```

Comment:
This initializer isn't necessary since it is always assigned in the constructor.